sparta/sparta/functional/RegisterBits.hpp

```diff
@@ -697,9 +725,10 @@ namespace sparta
 
     private:
 
-        std::array<uint8_t, 64> local_storage_; //!< Local storage
-        uint8_t               * local_data_  = nullptr; //!< Points to null if using remote data
-        const uint8_t         * remote_data_ = nullptr; //!< Remove data; points to local_data_ if no remote
-        const uint64_t          num_bytes_ = 0; //!< Number of bytse
+        std::array<uint8_t, 8>   local_storage_; //!< Local storage
+        std::unique_ptr<uint8_t> local_storage_alt_; //!< Alternative local storage when register size > 64B
```

Comment:
I think you meant `std::unique_ptr<uint8_t[]>` here.

Reply:
Oops sorry -- My “C” brain thought I could use that as a uint8_t pointer!